packages/router/src/RouterLink.ts

```diff
@@ -65,7 +65,9 @@ export interface RouterLinkProps extends RouterLinkOptions {
    */
   exactActiveClass?: string
   /**
-   * Value passed to the attribute `aria-current` when the link is exact active. Defaults to "page"
+   * Value passed to the attribute `aria-current` when the link is exact active.
+   *
+   * @default 'page'
```

Comment:
I suspect there's more JSDoc improvements that can be made
https://jsdoc.app/tags-param.html#optional-parameters-and-default-values
but that can be done later

Reply:
We actually use tsdoc which can infer many things from types, so don't worry!
Can you change this to `defaultValue` instead? (https://tsdoc.org/pages/tags/defaultvalue/)